wurst/util/PlayerUtils.wurst

```diff
@@ -0,0 +1,104 @@
+package PlayerUtils
+import LinkedList
+
+/**
+*   Credits to TriggerHappy for making PlayerUtils [vJass]
```

Comment:
Credits can be 1 line, an explanation about this package would be more useful.

Reply:
Will be adjusted and added